app/views/admin/budgets/_form.html.erb

```diff
@@ -63,6 +63,12 @@
                     class: "button hollow" %>
       <% end %>
 
+      <% if @budget.investments.where(winner: true).count > 0 %>
```

Comment:
Good! It's better to use scopes in views rather than `where` clauses
How about something like this:
`@budget.investments.winners.any?`